eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+id: eu.usdx.UltraStarDeluxe
+branch: master
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: true
+rename-desktop-file: ultrastardx.desktop
+rename-icon: ultrastardx
+command: ultrastardx
+
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+- --filesystem=~/.java/.userPrefs/PortMidi:ro
+- --persist=.ultrastardx
+
+cleanup:
+- /include
+- /lib/cmake
+- /lib/pkgconfig
+- "/lib/*.so"
+- "/lib/*.la"
+- setup_vars_opencv4.sh
+
+cleanup-commands:
+- rm -Rf /var/tmp/fpc
```

Comment:
this is not necessary. /var is never included.

Reply:
Ah yes, that line is a leftover from the time before switching to org.freedesktop.Sdk.Extension.freepascal